Competition2018/src/main/java/competition/subsystems/offboard/OffboardInterfaceSubsystem.java

```diff
@@ -146,6 +155,7 @@ public TargetCubeInfo getTargetCube() {
     @Override
     public void updatePeriodicData() {
         sendWheelOdomUpdate();
+        sendScoringPlacement();
```

Comment:
My understanding is that this data shouldn't change very often. Could we store the last placement data and only send when it changes? In the future we should probably make it possible to re-send it if the message is lost the first time, but for now that should work.